app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -339,12 +339,6 @@ export default {
       </div>
     </section>
 
-    <div v-if="clanIdSelected === ''" class="row flex-row text-center" style="margin-top: -25px; z-index: 0;">
```

Comment:
The original intent for this to be in a tag is because there is an `if` check here. If we are going to move the image into a class then we need to keep the check in some way. Otherwise we get the same image duplicated.

This page has two states. A global state, and a selected clan state. This background image should only appear in the `clanIdSelected === ''` which is the global state.

Here is a gif of me locally testing after inspecting a clan. When using local proxy, here's a test URL to reproduce the issue: http://localhost:3000/league/5febc4e0b0124ed69f57e07e

Note in the video the image appears twice.


https://user-images.githubusercontent.com/15080861/103585170-767f1400-4e97-11eb-9d1d-daa40c52c674.mp4

In production you can repro expected behavior (https://codecombat.com/league/5febc4e0b0124ed69f57e07e):

https://user-images.githubusercontent.com/15080861/103585285-a4fcef00-4e97-11eb-8aae-0e59a231a78d.mp4

Reply:
Made another another class `free-to-get-start-bg` to show only if `clanIdSelected === ''`